src/main/java/subway/path/util/AgeGroup.java

```diff
@@ -0,0 +1,48 @@
+package subway.path.util;
+
+public enum AgeGroup {
+    BABY, CHILD, TEENAGER, ADULT;
```

Comment:
그러기 위해서는 enum을 기술할 때 값을 미리 제공해 주어야 합니다.
```suggestion
    BABY(1.0, 0), CHILD(0.5, 350), TEENAGER(0.2, 350), ADULT(0.0, 0);

    private final int deduction;
    private final double discountRate;

    AgeGroup(double discountRate, int deduction) {
        this.discountRate = discountRate;
        this.deduction = deduction;
    }
```

Reply:
넵! 말씀주신대로 이렇게 바꿔주니 더 직관적인 것 같아요!
원래 코드에서 각 값들을 상수값으로 만들어줘야하나 했는데 이렇게 바꾸면 상수값을 선언할 필요가 아예 없겠네요😅